sky/provision/kubernetes/utils.py

```diff
@@ -1017,13 +1021,15 @@ def get_ssh_proxy_command(
     else:
         ssh_jump_proxy_command_path = create_proxy_command_script()
         current_context = get_current_kube_config_context_name()
+        current_namespace = get_current_kube_config_context_namespace()
         ssh_jump_proxy_command = construct_ssh_jump_command(
             private_key_path,
             ssh_jump_ip,
             ssh_jump_user=constants.SKY_SSH_USER_PLACEHOLDER,
             proxy_cmd_path=ssh_jump_proxy_command_path,
             proxy_cmd_target_pod=k8s_ssh_target,
-            current_kube_context=current_context)
+            current_kube_context=current_context,
+            current_kube_namespace=current_namespace)
```

Comment:
Let's add a comment here to explain why we add both context and namespace
```suggestion
            # We embed both the current context and namespace to the SSH proxy
            # command to make sure SSH still works when the current context/namespace
            # is changed by the user.
            current_kube_context=current_context,
            current_kube_namespace=current_namespace)
```